gene_benchmark/descriptor.py

```diff
@@ -4,6 +4,57 @@
 
 import mygene
 import pandas as pd
+import requests
+
+
+def _gene_symbol_to_ensemble_ids(
```

Comment:
why does this function start with an under score, isn't that only for the functions inside a class?

Reply:
In indicates a private method. In this context, it is private to the module.